src/main/java/frc/robot/Robot.java

```diff
@@ -191,7 +199,59 @@ public void autonomousInit() {
 
   /** This function is called periodically during autonomous. */
   @Override
-  public void autonomousPeriodic() {}
+  public void autonomousPeriodic() {
+    RAVEN_BLINKIN_3.setBlink(BlinkinCalibrations.FIRE);
```

Comment:
This is cool, but it will not work how you think it will. This will result in the Blinkin always being AQUA, since that is the last color it is set to before the function finishes. You are creating new wait commands, but they aren't being scheduled, and they do not pause execution for a second. The roborio will still execute this whole function in 1/50 of a second.

Reply:
For now I would put this logic into its own command which we could then schedule to run during autonomous.